internal/service/advancedcluster/resource_advanced_cluster_test.go

```diff
@@ -1181,7 +1186,17 @@ func checkReplicaSetMultiCloud(name string, regionConfigs int) resource.TestChec
 	)
 }
 
-func configShardedOldSchemaMultiCloud(orgID, projectName, name string, numShards int, analyticsSize string) string {
+func configShardedOldSchemaMultiCloud(orgID, projectName, name string, numShards int, analyticsSize string, configServerManagementMode *string) string {
+	var rootConfig string
+	if configServerManagementMode != nil {
+		// valid values: FIXED_TO_DEDICATED or ATLAS_MANAGED (default)
```

Comment:
knit: can be the comment in just one line?

Reply:
IMO: More readable as it is 3 different conditions. But can change it.